ch_2/tuples/named_tuples.py

```diff
@@ -0,0 +1,62 @@
+# Exerice 1: Use namedtuple and write a Python program that takes the people list (below), and produces a table.
+from collections import namedtuple
+
+people = [
+    ("Donald", "Trump", 7.85),
+    ("Vladimir", "Putin", 3.626),
+    ("Jinping", "Xi", 10.603),
+]
+
+
+def table_of_peope(people):
+
+    Person = namedtuple("Person", ["last", "first", "time"])
```

Comment:
I think you should move this line outside your function so it's accessible by other functions. Otherwise, only this function can use it.